CMakeLists.txt

```
@@ -57,6 +61,7 @@ else()
 endif()
 
 target_link_libraries(abacus PRIVATE steinwurf::endian)
+target_link_libraries(abacus PUBLIC protobuf::libprotobuf)
```

```suggestion
target_link_libraries(abacus PRIVATE protobuf::libprotobuf)
```
Don't let protobuf be exposed